functional-tests/src/test/groovy/spec/Permit.groovy

```diff
@@ -21,10 +21,8 @@ class  PermitSpec extends GebReportingSpec {
         permitTab.tabSelect.click()
 
         then:"I should see a list of permit"
-        assert permitTab.permit_no[2].text() == 'BLAHPER-02'
-        assert permitTab.permit_no[3].text() == 'BLAHPER-01'
-        assert permitTab.permit_date[2].text() == '2003-03-03'
-        assert permitTab.permit_date[3].text() == '2002-02-02'
+        assert permitTab.permit_no.text() != ''
```

Comment:
@VivianCheungFW Updated tests to check that permit_no and premit_date are not null, instead of checking for a specific variable

As per our testing discussion, we no longer need to check for the presence of data, that will be handled at the unit test level with snapshots.